### ISSUE TITLE
ValidationError for startsAt and endsAt fields

### ISSUE THREAD

**Issue by orgito** [idealista/prom2teams#122]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked that your issue isn't already filled: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aidealista
    * Checked that there is not already provided the described functionality

### Description

promt2teams is trowing the exception bellow after receiving a valid alert from alertmanager:

```
2019-06-06 13:53:17,619 - flask.app - ERROR - Exception on /v2/ToolsSquadTestes [POST]
Traceback (most recent call last):
...
  File "/usr/local/lib/python3.6/site-packages/prom2teams/app/versions/v2/namespace.py", line 26, in post
    alerts = self.schema.load(request.get_json())
  File "/usr/local/lib/python3.6/site-packages/marshmallow/schema.py", line 707, in load
    postprocess=True,
  File "/usr/local/lib/python3.6/site-packages/marshmallow/schema.py", line 867, in _do_load
    raise exc
marshmallow.exceptions.ValidationError: {'alerts': {0: {'endsAt': ['Not a valid date.'], 'startsAt': ['Not a valid date.']}}}
```

### Steps to Reproduce

1 - Alertmanager sends an alert including valid RFC3339 dates. But the format does not conform with the validation message_schema.py:

Excerpt of a sample alert:
```json
{
...
      "startsAt": "2019-06-05T16:22:28.29468897Z",
      "endsAt": "0001-01-01T00:00:00Z",
...
}
```

message_schema.py defines those fields as:
```python
startsAt = fields.Date('%Y-%m-%dT%H:%M:%S.%fZ')
endsAt = fields.Date('%Y-%m-%dT%H:%M:%S.%fZ')
```

The problem here is the use of `%f` for the second fraction. This limits the fraction to microsseconds. But higher precision is valid in RFC3339. It also rejects valid dates that do not have the fractions.

A quickfix is to change the fields to:
```python
startsAt = fields.DateTime()
endsAt = fields.DateTime()
```

But this will accept other date formats too.

**Expected behavior:** prom2teams shoud accept any RFC3339 valid format for startsAt and endsAt

**Actual behavior:** prom2teams is trowing an exception when the second fraction is higher precision than microseconds or when there is no second fraction.

**Reproduces how often:** Every time one of startsAt and endsAt contains a higher precision fractions for seconds or does not contains the fractions.

### Versions

python 3.6.8
alertmanager 0.17.0
prometheus 2.10.0
prom2teams 2.3.1 (installed via pip)
marshmallow 3.0.0rc6 (installed via pip as prom2teams dependecy)

**Comment by miguel-chacon** [comment]
We'll take a look ASAP, thanks @orgito

**Comment by orgito** [comment]
Thank you!

**Comment by prsshini** [comment]
Is this issue resolved? I have the latest prom2teams 3.2.2 which throws the same error.. 

When I post the message without startsAt inthe payload Json, I get the error 
curl -X POST $url -d '[{"status": "firing","labels": {"alertname": "subha-1021pm","service": "curl","severity": "critical","instance": "0"},"annotations": {"summary": "This is a summary","description": "This is a description."},"generatorURL": "http://prometheus.int.example.net/<generating_expression>"}]'


Error Message:
 marshmallow.exceptions.ValidationError: {'alerts': {0: {'startsAt': ['Not a valid datetime.']}}}
Jul 23 05:24:17 ip-172-18-0-114 prom2teams[11629]: 2021-07-23 05:24:17,464 - werkzeug - INFO - 127.0.0.1 - - [23/Jul/2021 05:24:17] "POST /v2/Connector HTTP/1.1" 500 -



When I post it with the startsAt in the payload, I can send the message successfully.
curl -X POST $url -d '[{"status": "firing","labels": {"alertname": "subha-955pm","service": "curl","severity": "critical","instance": "0"},"annotations": {"summary": "This is a summary","description": "This is a description."},"generatorURL": "http://prometheus.int.example.net/<generating_expression>","startsAt": "2020-07-23T01:05:36+00:00"}]'
